Index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Page Title</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.2.1/css/fontawesome.min.css" integrity="sha384-QYIZto+st3yW+o8+5OHfT6S482Zsvz2WfOzpFSXMF9zqeLcFV0/wlZpMtyFcZALm" crossorigin="anonymous">
+</head>
+<body> 
+    <div class="wrapper">
+      <section class="section1">
+       <header class="toolbar">           
+            <span id="logo">My Logo</span> 
+            <span id="switch">               
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>             
+            </span>                  
+       </header>
+       </section>
+       <section class="section2">
+        <div id="greetings">I’m Alexandra. Glad to see you!</div>
```

Comment:
Kindly remove the `div` tag here. Instead, let's make this a heading by using an `h1` tag since this is your main title for the whole application.
If you want that google (and other search engines) to know about a specific topic, in this case, you, use an `h1` tag.